src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -8,6 +8,13 @@
     <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(TargetOS)' == 'Browser'">true</DebuggerSupport>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <IsHighAotMemoryUsageTest>true</IsHighAotMemoryUsageTest> <!-- to avoid OOMs with source generation in wasm: https://github.com/dotnet/runtime/pull/60701 -->
+
+    <!-- Undo once the repo moves to a sufficiently high-enough version for file-scoped types -->
+    <MicrosoftCodeAnalysisVersion>4.4.0-1.22356.23</MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>4.4.0-1.22356.23</MicrosoftCodeAnalysisCSharpVersion>
```

Comment:
Why do we need to define the resto of the versions as opposed to just `MicrosoftCodeAnalysisVersion` ? That is the only one I see used in this file. The other one used is `CompilerPlatformTestingVersion`, should that be moved to match CodeAnalysis version as well?

Reply:
We may not.  I just overrode all the Microsoft.CodeAnalysis.* ones that were defined in Version.props in order to keep them consistent.

Reply:
Right, the properties only get used if a PackageReference uses them, and this project only uses the first one, and the `CompilerPlatformTestingVersion` so I think we should just remove the unused ones and perhaps add the `CompilerPlatformTestingVersion` to match.